### ISSUE TITLE
Upgrade from 0.7.0 to 0.8.2 killing application broker

### ISSUE THREAD

**Issue by akrausesap** [kyma-project/kyma#3406]
**Description**

Hi Colleagues,

the upgrade of a 0.7.0 cluster to 0.8.2 caused the application broker to crash with OOM exceptions. I know the scenario is not "supported" but maybe the logs are interesting for you.

Traces:

[pods.txt](https://github.com/kyma-project/kyma/files/3018248/pods.txt)
[logs-application-operator-0.txt](https://github.com/kyma-project/kyma/files/3018249/logs-application-operator-0.txt)
[describe-application-operator-0.txt](https://github.com/kyma-project/kyma/files/3018250/describe-application-operator-0.txt)

Cheers

Andreas

**Expected result**

N/A

**Actual result**

N/A

**Steps to reproduce**

```
kubectl apply -f https://github.com/kyma-project/kyma/releases/download/0.8.2/kyma-installer-cluster.yaml
kubectl label installation/kyma-installation action=install
```

**Troubleshooting**

N/A

**Comment by Szymongib** [comment]
Did the update finish without issues? It seems that the Application Operator did not even get updated.

**Comment by akrausesap** [comment]
no idea. unfortunately I don't have access to the cluster anymore.

**Comment by crabtree** [comment]
We have a similar problem and it was addressed here #3682.
